nio/block/mixins/retry/strategies/tests/test_exponential.py

```diff
@@ -1,109 +1,97 @@
 from unittest.mock import patch
-from nio import Block
-from nio.block.mixins.retry.retry import Retry
-from nio.testing.block_test_case import NIOBlockTestCase
+from nio.util.logging import get_nio_logger
+from nio.block.mixins.retry.strategies.exponential import ExponentialBackoff
+from nio.testing import NIOTestCase
 
 
-class ExponentialBackoffBlock(Retry, Block):
-    """ An example of a block that uses a Exponential Backoff strategy """
-    pass
+class TestExponentialBackoff(NIOTestCase):
 
-
-class TestExponentialBackoff(NIOBlockTestCase):
-
-    def assert_next_retry_sleeps_for(self, block, num_seconds):
+    def assert_next_retry_sleeps_for(self, strat, num_seconds):
         """Make sure that given a failure the block will sleep for some time"""
         sleep_path = 'nio.block.mixins.retry.strategies.exponential.sleep'
         with patch(sleep_path) as sleep:
-            block._backoff_strategy.request_failed(Exception())
-            if block._backoff_strategy.should_retry():
-                block._backoff_strategy.wait_for_retry()
+            strat.request_failed(Exception())
+            if strat.should_retry():
+                strat.wait_for_retry()
             sleep.assert_called_once_with(num_seconds)
 
     def test_default(self):
         """Test the default behavior of the exponential backoff"""
-        block = ExponentialBackoffBlock()
-        self.configure_block(block, {
-            "retry_options": {
-                "strategy": "exponential"
-            }
-        })
+        strat = ExponentialBackoff(
```

Comment:
These tests were previously creating a Block instance which is a bit out of scope given that this should be a test specific to the strategy. So I have adjusted them to instantiate the strategy directly, these tests shouldn't care much about blocks